dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -396,8 +397,8 @@ private static void manualPublish(List<String> args) {
             final String namespace = reqVal(args, "--namespace");
             final String gitURL = reqVal(args, "--git-url");
 
-            final String dockerfilePath = optVal(args, "--dockerfile-path","/Dockerfile");
-            final String cwlPath = optVal(args, "--cwl-path","Dockstore.cwl");
+            final String dockerfilePath = optVal(args, "--dockerfile-path", "/Dockerfile");
+            final String cwlPath = optVal(args, "--cwl-path", "Dockstore.cwl");
```

Comment:
This looks like a potential typo, I think the default here should be `/Dockstore.cwl` not `Dockstore.cwl`